pandas/core/ops/__init__.py

```diff
@@ -265,7 +265,7 @@ def _get_opstr(op):
         rtruediv: "/",
         operator.floordiv: "//",
         rfloordiv: "//",
-        operator.mod: None,  # TODO: Why None for mod but '%' for rmod?
+        operator.mod: "%",
```

Comment:
do we have tests where this makes a difference?  maybe this lets us use numexpr and performance is affected?

Reply:
Yeah, I wasn't sure about this one. Thought I give it a try and see if any of the CI hooks is failing. The 2 failing ones in `pandas-dev.pandas (Linux py37_locale)` look unrelated though. I'll rebase and try again...

Reply:
I undid the change now and handle the symbol conversion in the test case itself. The CI is still failing though...

Reply:
@jbrockmendel CI is green now, so all tests seem to be fine with this change.
@jreback do you have any concerns about it?

Reply:
looks ok, original issue is that this looks like a format string to python